src/Components/HomePage.js

```diff
@@ -2,53 +2,129 @@ import React from 'react';
+              </Text>
             </View>
-          </ImageBackground>
-        </TouchableHighlight>
-        <TouchableHighlight
-          onPress={() => this.props.navigation.navigate('DrillListPage', { type: DrillTypes.FITNESS })}
-          style={styles.menuItem}
-        >
-          <ImageBackground source={fitness} style={styles.image}>
-            <View style={styles.wrapper}>
-              <Text style={styles.text}>Fitness drills</Text>
+          </View>
+        </ImageBackground>
+      </TouchableHighlight>
+      <TouchableHighlight
+        onPress={() => props.navigation.navigate('DrillListPage', { type: DrillTypes.FITNESS })}
+        style={styles.menuItem}
+      >
+        <ImageBackground source={bodyweight} style={styles.image}>
+          <View style={styles.wrapperCenter}>
+            <Text style={styles.titleCenter}>BODYWEIGHT ONLY</Text>
+            <View style={styles.description}>
+              <Text style={styles.textFitness}>
+                Push your strenght and improve muscle tone over the season - all without weights
+              </Text>
             </View>
-          </ImageBackground>
-        </TouchableHighlight>
-        <TouchableHighlight onPress={() => this.props.navigation.navigate('ProgramListPage')} style={styles.menuItem}>
-          <ImageBackground source={huddle} style={styles.image}>
-            <View style={styles.wrapper}>
-              <Text style={styles.text}>Training programs</Text>
+          </View>
+        </ImageBackground>
+      </TouchableHighlight>
+      <TouchableHighlight
+        onPress={() => props.navigation.navigate('DrillListPage', { type: DrillTypes.FITNESS })}
+        style={styles.menuItem}
+      >
+        <ImageBackground source={gymstrong} style={styles.image}>
+          <View style={styles.wrapperCenter}>
+            <Text style={styles.titleCenter}>GYM STRONG</Text>
+            <View style={styles.description}>
+              <Text style={styles.textFitness}>
+                Build full-body strenght with a focus on weight training over the season
+              </Text>
             </View>
-          </ImageBackground>
-        </TouchableHighlight>
-        <View style={styles.editorLink}>
-          <Button
-            onPress={() => this.props.navigation.navigate('AnimationEditorPage')}
-            text="Animation Editor"
-            buttonLight="true"
-          />
-        </View>
-      </View>
-    );
-  }
+          </View>
+        </ImageBackground>
+      </TouchableHighlight>
+    </View>
+  );
+};
+
+export default function HomePage(props) {
+  return (
+    <Tab.Navigator
+      initialRouteName="Frisbee"
+      activeColor={theme.COLOR_PRIMARY}
+      inactiveColor={theme.COLOR_SECONDARY}
+      barStyle={{ backgroundColor: theme.BACKGROUND_COLOR }}
+    >
+      <Tab.Screen
+        name="Frisbee"
+        component={HomeScreen}
+        options={{
+          tabBarLabel: 'Home',
```

Comment:
Is it really `Home` and `Fitness`? Or `Frisbee` and `Fitness`?

Reply:
It's Frisbee and Fitness But I hav'nt found a frisbee icon yet